manuscript/the-meat.txt

```diff
@@ -513,21 +513,21 @@ But the `getPreviousYearFragment` function is more complex:
 
         return fragment;
     },
+**
+First, we get the list of years. If it’s empty or the current year is 2012 (the first in our dataset), we return an empty fragment.
 
-First we get the list of years. If it’s empty or the current year is 2012 (the first in our dataset), we return an empty fragment.
-
-Otherwise we get last year’s data with `this.getAllDataByYear()`, then compare the number of entries with this year. We return a fragment that’s either “X% **more** than the year before” or “X% **less** than the year before”.
+Otherwise, we get last year’s data with `this.getAllDataByYear()`, then compare the number of entries with this year. We return a fragment that’s either “X% **more** than the year before” or “X% **less** than the year before”.
 
 Now we’ve just got to use these two fragments in the render method.
```

Comment:
..First, we get the list of years. If it’s empty or the current year is 2012 (the first in our dataset), we return an empty fragment.

In any other case, we get a look at last year’s data with `this.getAllDataByYear()`. We can then compare the number of entries with this year and return a fragment that’s either “X% **more** than the year before” or “X% **less** than the year before”.